.github/workflows/ci.yml

```diff
@@ -32,18 +26,16 @@ jobs:
           override: true
           target: wasm32-unknown-unknown
 
-    #   - uses: actions/cache@v2
-    #     with:
-    #       path: |
-    #         ~/.cargo/registry
-    #         ~/.cargo/git
-    #         target
-    #       key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
-    #       restore-keys: |
-    #         ${{ runner.os }}-cargo-
+      - name: Cache darwinia target
+        uses: actions/cache@v2
+        with:
+          path: target
+          key: ${{ runner.os }}-darwinia
 
       - name: Overall test
-        run: cargo test
+        env:
+          RUST_TOOLCHAIN: nightly-2020-10-06
```

Comment:
Is this line will affect the sr-tool?

Reply:
no, srtool only build release. is there
https://github.com/fewensa/darwinia/blob/features%2Fci/.github/workflows/release.yml#L36

Reply:
So nightly is fine.

In #636

Reply:
yeah, in .github/workflows/ci.yml use nightly is fine, but when you create a new tag, will run .github/workflows/release.yml

you can see that
https://github.com/darwinia-network/darwinia/runs/2146405787?check_suite_focus=true#step:5:37
currently release build use `nightly-2020-10-06` so when you push a new tag, this version may not work.


your change not effect commit to push master, but your code may not support `nightly-2020-10-06` (currently code), and `nightly-2020-10-27` (this pr, srtool use this version). 


in this pr, .github/workflows/release.yml build docker images can use `nightly` version, but paritytech/srtool only support `nightly-2020-10-27`

Reply:
Will check.